spec/services/twilio_service_spec.rb

```diff
@@ -62,18 +63,19 @@
       }
 
       before do
-        allow(TwilioService).to receive(:fetch_attachment).with(image_url).and_return({
+        @twilio_service = TwilioService.new(:gyr)
+        allow(@twilio_service).to receive(:fetch_attachment).with(image_url).and_return({
                                                                                         filename: "a_real_image.jpg",
                                                                                         body: "image body"
                                                                                       })
-        allow(TwilioService).to receive(:fetch_attachment).with(pdf_url).and_return({
+        allow(@twilio_service).to receive(:fetch_attachment).with(pdf_url).and_return({
```

Comment:
i'm unfamiliar with using instance variables in rspec specs—typically we'd put something like this in a `let` block (not to say it's better, just that it's the practice). happy to discuss!